provider/pkg/await/await.go

```diff
@@ -196,6 +196,10 @@ func Creation(c CreateConfig) (*unstructured.Unstructured, error) {
 				}
 				outputs, err = client.Patch(
 					c.Context, c.Inputs.GetName(), types.ApplyPatchType, objYAML, options)
+
+				if errors.IsConflict(err) {
+					err = fmt.Errorf(`use the "pulumi.com/patchForce" annotation if you want to overwrite the existing values: %w`, err)
```

Comment:
I initially wondered if it would read better with the advice at the end; but, I think it's better to have it as you'd done, with the important information ("there's a problem, here's how to fix it") first.